### PR TITLE
Intel Blog - Part 2

### PR DESCRIPTION
**Pull request by mfuntowicz**

### CODE REVIEW

**Review by jeffboudier — APPROVED** [review]
Proofread the post, mostly graphs should be more readable, and the conclusion needs to be finished.

File: bert-cpu-scaling-part-2.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+    <a href="/michaelbenayoun">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1615890856777-6047a3315da6ba4b1dfb9e18.png?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>michaelbenayoun</code>
+            <span class="fullname">Michael Benayoun</span>
+        </div>
+    </a>
+</div>
+
+# Scaling up BERT-like model Inference on modern CPU  - Part 2
```
Comment (julien-c) [inline bert-cpu-scaling-part-2.md:36]: can we keep the same order as on other blogposts? i.e. title, then "Published on", then "Update on GitHub", then "author faces"

File: bert-cpu-scaling-part-2.md [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+      display: block;
+      margin: 0 auto;
+  }
+
+  figure {
+      text-align: center;
+      display: table;
+      max-width: 85%; /* demo; set some amount (px or %) if you can */
+      margin: 10px auto; /* not needed unless you want centered */
+  }
```
Comment (julien-c) [inline bert-cpu-scaling-part-2.md:12]: you should use Tailwind instead

File: bert-cpu-scaling-part-2.md [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+</style>
+
```
Comment (julien-c) [inline bert-cpu-scaling-part-2.md:14]: need a metadata/yaml block too

File: bert-cpu-scaling-part-2.md [hunk 4]
```diff
@@ -0,0 +1,536 @@
...
+            <span class="fullname">Morgan Funtowicz</span>
+        </div>
+    </a>
+    <a href="/michaelbenayoun">
+        <img class="avatar avatar-user" src="https://aeiljuispo.cloudimg.io/v7/https://s3.amazonaws.com/moonup/production/uploads/1615890856777-6047a3315da6ba4b1dfb9e18.png?w=200&h=200&f=face" title="Gravatar">
+        <div class="bfc">
+            <code>michaelbenayoun</code>
+            <span class="fullname">Michael Benayoun</span>
+        </div>
+    </a>
```
Comment (julien-c) [inline bert-cpu-scaling-part-2.md:33]: would it be relevant to add co-authors from Intel, or not really?

File: bert-cpu-scaling-part-2.md (hunk 1 above)
Comment (mfuntowicz) [reply bert-cpu-scaling-part-2.md:36]: I copy/pasted this from the previous blog I did, should I update there too?

File: bert-cpu-scaling-part-2.md (hunk 4 above)
Comment (mfuntowicz) [reply bert-cpu-scaling-part-2.md:33]: I would personally say not really, @jeffboudier @michaelbenayoun wdyt?

File: bert-cpu-scaling-part-2.md (hunk 1 above)
Comment (julien-c) [reply bert-cpu-scaling-part-2.md:36]: yes please!

File: bert-cpu-scaling-part-2.md (hunk 2 above)
Comment (julien-c) [reply bert-cpu-scaling-part-2.md:12]: (happy to have someone help if needed!)

File: bert-cpu-scaling-part-2.md (hunk 3 above)
Comment (julien-c) [reply bert-cpu-scaling-part-2.md:14]: at least w/ the title

File: bert-cpu-scaling-part-2.md (hunk 4 above)
Comment (julien-c) [reply bert-cpu-scaling-part-2.md:33]: and what about @echarlaix ? :)

Comment (jeffboudier) [reply bert-cpu-scaling-part-2.md:28]: best not to include Intel people, the only way we can publish is if we own the results 100%

**Pull request merged**